test/e2e/data/infrastructure-docker/v1beta1/bases/cluster-with-topology.yaml

```
@@ -17,11 +17,15 @@ spec:
     version: "${KUBERNETES_VERSION}"
     controlPlane:
       metadata: {}
+      nodeDrainTimeout: "30s"
+      nodeDeletionTimeout: "30s"
       replicas: ${CONTROL_PLANE_MACHINE_COUNT}
     workers:
       machineDeployments:
       - class: "default-worker"
         name: "md-0"
+        nodeDrainTimeout: "30s"
+        nodeDeletionTimeout:: "30s"
```

```suggestion
        nodeDeletionTimeout: "30s"
```
This should fix the broken test.

---

Corrected. I could repro the NodeDrainTimeout Issue on a kind cluster. Trying to try out the fix on the kind cluster.